### PR TITLE
7540 NameRequest Dashboard Btn

### PR DESCRIPTION
**Pull request by cameron-freshworks**
*Issue #:*
bcgov/entity#7540

*Description of changes:*

* Added Name Request btn to Dashboard.

* Open NR App in a new tab.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/components/auth/manage-business/EntityManagement.vue [hunk 1]
```diff
@@ -23,7 +33,7 @@
                 <span>{{ $t('addExistingBtnLabel') }}</span>
               </v-btn>
             </template>
-            <v-list>
+            ss<v-list>
```
Comment (severinbeauvais) [inline auth-web/src/components/auth/manage-business/EntityManagement.vue:36]: Oops?

File: auth-web/src/components/auth/manage-business/EntityManagement.vue [hunk 2]
```diff
@@ -380,6 +399,10 @@ export default class EntityManagement extends Mixins(AccountChangeMixin, NextPag
   .view-header {
     justify-content: space-between;
 
+    #add-name-request-btn {
+      font-weight: normal;
```
Comment (severinbeauvais) [inline auth-web/src/components/auth/manage-business/EntityManagement.vue:403]: There's a Vuetify class for this if you don't want this class her.

File: auth-web/src/components/auth/manage-business/EntityManagement.vue (hunk 1 above)
Comment (cameron-freshworks) [reply auth-web/src/components/auth/manage-business/EntityManagement.vue:36]: Oops!

File: auth-web/src/components/auth/manage-business/EntityManagement.vue (hunk 2 above)
Comment (cameron-freshworks) [reply auth-web/src/components/auth/manage-business/EntityManagement.vue:403]: oh nice, i see it. Font weight regular? 
I'd prefer the helper class. Updated!

**Comment by cameron-freshworks** [comment]
There is a failing test, but it is must have been pre-existing as it is unrelated to my update.

**Review by severinbeauvais — APPROVED** [review]

File: auth-web/src/components/auth/manage-business/EntityManagement.vue [hunk 3]
```diff
@@ -262,6 +273,14 @@ export default class EntityManagement extends Mixins(AccountChangeMixin, NextPag
     return LaunchDarklyService.getFlag(LDFlags.EnableMandatoryAddress) || false
   }
 
+  // open Name Request
+  private goToNameRequest (): void {
```
Comment (sumesh-aot) [inline auth-web/src/components/auth/manage-business/EntityManagement.vue:277]: is opening in a new window a requirement ? If you open in new window the session won't get sync'ed as the session storage values won't be present in new window.

Comment (cameron-freshworks) [reply auth-web/src/components/auth/manage-business/EntityManagement.vue:277]: hmm ok, i was curious about this. 
The ticket DOES call out opening in a new tab explicitly, but if this is going to create a sync issue then perhaps we keep the same tab?

Comment (sumesh-aot) [reply auth-web/src/components/auth/manage-business/EntityManagement.vue:277]: You may need to discuss with the UX team and convince this issue :)

Comment (severinbeauvais) [reply auth-web/src/components/auth/manage-business/EntityManagement.vue:277]: Good catch, Sumesh. That was why NR UI was recently changed to open in the same window.

**Review by sumesh-aot — APPROVED** [review]

File: auth-web/src/components/auth/manage-business/EntityManagement.vue (hunk 3 above)
Comment (cameron-freshworks) [reply auth-web/src/components/auth/manage-business/EntityManagement.vue:277]: Updated! Will post comment in ticket for UX, thanks!

**Review by shabeeb-aot — APPROVED** [review]

**Pull request merged**